cookbook/Tip-to-use-statics-files-js-css.md

```diff
@@ -0,0 +1,27 @@
+# Static Files
+
+## How let the Yesod know about your file
+
+Yesod build the static files to use inside the project.
+
+Exemple We have one file in project:
+
+	/static/css/my_css_file.css
+
+When Yesod build the libraries, a variable with name css_my_css_file_css is created to use inside Handlers.
+
+If you don't build, he no know about the file and when you use css_my_css_file_css, it no exists.
```

Comment:
Probably this can be replace with something like "The project needs to be rebuilt again to bring `css_my_css_file_css` under the scope" ?